config/config.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+
+module.exports = {
+  development: {
+    dialect: 'postgres',
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME, 
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+  },
+  test: {
+    use_env_variable: 'DATABASE_URL_TEST',
+    dialect: 'postgres'
+  },
+  production: {
+    use_env_variable: 'DATABASE_URL',
+    dialect: 'postgres'
+  },
+};
```

Comment:
It would be fine if the definition of the `development` environment is consistent with what you have in `test` and `production`.